apisix/core.lua

```diff
@@ -19,7 +19,9 @@ local local_conf = require("apisix.core.config_local").local_conf()
 
 local config_center = local_conf.apisix and local_conf.apisix.config_center
                       or "etcd"
+local etcd_version = local_conf.etcd.version == "v3" and "_v3" or ""
```

Comment:
IMO, we should ONLY supports etcd v3, not etcd v2.

Reply:
Currently, the implementation is to use v2 as the default, and user would change to v3 if needed. Of course we can use v3 as default.

ping @nic-chen @membphis maybe we need to discuss about it